lib/PHPExif/Exif.php

```diff
@@ -969,10 +969,10 @@ public function getAltitude() : float|false
     /**
      * Sets the altitude value
      *
-     * @param string $value
+     * @param float|false $value
      * @return \PHPExif\Exif
      */
-    public function setLongitude(string $value) : Exif
+    public function setLongitude(float|false $value) : Exif
```

Comment:
Personally, I am not a big fan of using `false` to indicated the absence of data. I would prefer using `null` for that, i.e. `?float` in this specific case. However, I cannot estimate where a `false` value is already been used else and how much work it would be to replace it by `null` everywhere.

Reply:
Another reason to go for `null|float` is that it allows to use `??` instead of the Elvis operator. That for me alone is a big plus.